Sources/GRPCCodeGen/Internal/Translator/IDLToStructuredSwiftTranslator.swift

```diff
@@ -49,6 +49,40 @@ struct IDLToStructuredSwiftTranslator: Translator {
   }
 }
 
+extension IDLToStructuredSwiftTranslator {
+  private func translateImport(
+    dependency: CodeGenerationRequest.Dependency
+  ) throws -> ImportDescription {
+    var importDescription = ImportDescription(moduleName: dependency.module)
+    if let item = dependency.item {
+      if let matchedKind = ImportDescription.Kind.allCases.first(where: {
+        "\($0)" == item.kind.value.description
+      }) {
```

Comment:
`ImportDescription.Kind` has a raw value of `String`. This means that it provides an `init?(rawValue:)` initialiser. You can take advantage of it to avoid iterating over all cases as you're doing here - if the provided raw value doesn't match an enum case, the initialiser will return nil:
```suggestion
      if let matchedKind = ImportDescription.Kind(rawValue: item.kind.value.description) {
```